### ISSUE TITLE
Is it possible to train PICK-pytorch to detect table and all its line-items?

### ISSUE THREAD

**Issue by locomotiivo** [wenwenyu/PICK-pytorch#111]
Hi wenwenyu, thank you for an amazing code. 
I have been experimenting around with the code, and found out the training dataset can be adjusted for extractions of different informations.

However, there's one thing I am stuck on, and that is training the model to detect tables and its contents as well. 
I want the customized model to be able to not only detect the header data, but also list all the table line-item.
From what I understand of the code, it seems possible to train it to detect table contents as well, but I don't know how I should set the training data's entities/labels, especially for tables with more than one line-items.

Any help or tip would be greatly apppreciated, thanks :)

**Comment by ziodos** [comment]
it would be better to check a model for detecting table shape, and then you can parse the content and arrange it, I think the PICK model would be more efficient when you want to extract unstructured data and not tabular ones.
